packages/core/lib/enums/ASTTypeEnum.ts

```diff
@@ -0,0 +1,11 @@
+enum ASTTypeEnum {
```

Comment:
we need to share this between both packages, so it can't be in core

Reply:
moved the type to the new shared package